apps/mobile/src/libs/components/on-leave-modal/on-leave-modal.tsx

```diff
@@ -20,14 +20,16 @@ type ModalButtonConfiguration = {
 
 type Properties = {
 	buttonsConfiguration: ModalButtonConfiguration[];
+	description: string;
 	isVisible: boolean;
 	onBackdropPress: () => void;
 };
 
 const ICON_SIZE = 70;
 
-const SignOutModal: FC<Properties> = ({
+const OnLeaveModal: FC<Properties> = ({
```

Comment:
and file name as well
```suggestion
const ConfirmationModal: FC<Properties> = ({
```